src/lib.rs

```diff
@@ -2382,7 +2405,7 @@ impl Build {
         enum ArchSpec {
             Device(&'static str),
             Simulator(&'static str),
-            Catalyst(&'static str),
+            Catalyst(#[allow(dead_code)] &'static str),
```

Comment:
This is only here because the new nightly versions correctly detect that this field is never used, which causes the CI to fail.